hack/ci/lvm-setup.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# This file is part of MinIO Direct CSI
+# Copyright (c) 2021 MinIO, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+set -ex
+
+declare -a loops=("loop0" "loop1" "loop2" "loop3")
+for loop in "${loops[@]}"; do
+    dd if=/dev/zero of=/tmp/${loop}.img bs=100M count=10
+    sudo losetup -fP /tmp/${loop}.img
```

Comment:
`losetup -f` will assign any free loopback device i.e. you can't assume it will be `/dev/loop0` for first run. you could avoid `-P` usage as the file is always fresh.